collect_app/src/main/java/org/odk/collect/android/formentry/questions/AudioVideoImageTextLabel.java

```diff
@@ -143,7 +142,20 @@ public void setAudio(String audioURI, AudioHelper audioHelper) {
     }
 
     public void setImage(@NonNull File imageFile) {
-        setupImage(imageFile);
+        if (imageFile.exists()) {
+            imageView.layout(0, 0, 0, 0);
+
+            Glide.with(this)
+                    .load(imageFile)
+                    .centerInside()
+                    .into(imageView);
+
+            imageView.setVisibility(VISIBLE);
+            imageView.setOnClickListener(this);
+        } else {
+            missingImage.setVisibility(VISIBLE);
+            missingImage.setText(getContext().getString(R.string.file_missing, imageFile));
```

Comment:
@lognaturel maybe it's time to get rid of those error messages? Glide can display a special image that will be used if something is wrong so we could have such an error/missing file image?

Reply:
I think using a missing image file is a good idea and will make it more obvious that something is wrong.

The advantage of the message is that it lets the user know where the image is expected to be and what its file name is. How about leaving this message and adding the missing image? Or is there a strong reason to get rid of it?

Reply:
Ok I'll file a separate issue for that.